swan-lake/learn-the-platform/ballerina-persist/persist-cli.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: ballerina-persist-documentation-left-nav-pages-swanlake
+title: Persist CLI tool
+description: The sections gives details on the Persist CLI tool.
+keywords: ballerina, programming language, ballerina packages, persist, persist cli tool, persist init, persist generate, persist migrate
+permalink: /learn/ballerina-persist/persist-cli/
+active: persist_cli
+intro: One of the main components of the `bal persist` feature is the CLI tool. This tool is used to initialize the project with bal persist and generate the required files.
```

Comment:
```suggestion
intro: One of the main components of the bal persist feature is the CLI tool. This tool is used to initialize the project with the bal persist command and generate the required files.
```
Backticks will not be rendered when added in the front matter. It will be fine to not have the backticks in the first sentence as it refers directly to the feature.  Shall we move the second sentence out of the intro as it refers to the command, which should be added within backticks for it to render properly?